Makefile

```diff
@@ -1,12 +1,16 @@
 default: lintf
 
 PROTO_FILES=./protos/node/node.proto \
+
+ 			./protos/guard/guard.proto \
+ 			./protos/shared/serverstatus.proto \
```

Comment:
I'd suggest change folder shared to the same name with the package, since node, ledger and potentially others are also "shared" yet they cannot put under the same folder.

Plus, can we change the package/folder name to health? -- Since we already have status-server, and I'm afraid serverstatus might confuse people.

Reply:
or some other name to distinguish status-server

Reply:
The idea shared here is try to say this service will be shared among all other proto. For the name, I think it over, the accurate name is RuntimeInfo, but looks weird,  your idea?

Reply:
Java Spring has a bean for these kind of work, it use /info to show this kind of information, but info is pretty big term.

Reply:
Finally changed the name to RuntimeInfo as we discussed.